package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@eppo/react-native-sdk",
-  "version": "3.7.2",
+  "version": "3.7.3-alpha1",
```

Comment:
I usually use `yarn version --prepatch --preid alpha` to increment to a pre-patch alpha release and it auto-inserts a `.`, i.e. `3.7.3-alpha.1`

Reply:
oh good call